lib/iris/quickplot.py

```diff
@@ -13,8 +13,10 @@
 
 """
 
+import packaging
```

Comment:
Currently failing with an `AttributeError` (e.g. https://travis-ci.org/github/SciTools/iris/jobs/724152197#L1475)
Suggest switching to `from packaging import version`, and all the follow-ons from that. I prefer this syntax anyway since it's safe against such `AttributeError`s.

Reply:
@trexfeathers That'll teach me to push without running if first :laughing: